docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>maybe link directly Sinkhorn with the solver module?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>